chrome/content/zotero/xpcom/translation/translator.js

```diff
@@ -29,10 +29,10 @@ var TRANSLATOR_REQUIRED_PROPERTIES = ["translatorID", "translatorType", "label",
 // Properties that are preserved if present
 var TRANSLATOR_OPTIONAL_PROPERTIES = ["browserSupport", "minVersion", "maxVersion",
                                       "inRepository", "configOptions", "displayOptions",
-                                      "hiddenPrefs"];
+                                      "hiddenPrefs", "itemType"];
```

Comment:
I'm not entirely sure what happened to internal representation of translators going from 4.0 to 5.0, but it seems that they used to be represented and passed around as objects and now there's a `Zotero.Translator` class, with a constructor, which limits the types of fields available on construction.

`itemType` is a translator property, which wasn't included and is set [here](https://github.com/adomasven/zotero/blob/905d94c681ceff92c071d7e41a117282bf2042ce/chrome/content/zotero/xpcom/translation/translate.js#L1437-L1437). Translators get passed around between background and injected scripts and they get serialised and deserialised [here](https://github.com/adomasven/zotero-connectors/blob/feature/zotero5-translator-code-support/src/common/messages.js#L78-L103), so this property needs to be supported by `Zotero.Translators.prototype.init()`.